src/mock/person/personsokMock.ts

```diff
@@ -30,7 +35,7 @@ function getPersonsokResponse(): PersonsokResponse {
     const postadresse = vektetSjanse(faker, 0.5) ? getPostadresse() : null;
     const bostedsadresse = vektetSjanse(faker, 0.5) ? getBostedsadresse() : null;
     const brukerinfo = vektetSjanse(faker, 0.5) ? getBrukerinfo() : null;
-    const utenlandskID = vektetSjanse(faker, 0.5) ? getUtenlandskID() : null;
+    const utenlandskID = null;
```

Comment:
Kunne ikke dette vært `vektetSjanse(faker, 0.5) ? getUtenlandskID() : null` også slipper man en helt egen funksjon for å lage personer med utenlandskid?

Reply:
Denne endringen ble gjort for å matche backend da noen av elementene vil være `null` når vi bruker søk på `utenlandskID` men de skal ha en verdi hvis vi ikke gjør søk med `utenlandskID`